tests/src/Traits/SchemaPrinterTrait.php

```diff
@@ -19,7 +19,7 @@ trait SchemaPrinterTrait {
    * @return string
    *   The printed version of the schema.
    */
-  protected function getPrintedSchema($server = NULL) {
+  protected function getPrintedSchema(\Drupal\graphql\Entity\ServerInterface $server = NULL) {
```

Comment:
similar here, let's use just `ServerInterface` and make sure namespace is added to the top of the file:
```
use Drupal\graphql\Entity\ServerInterface;
```